### ISSUE TITLE
inputNumber buttons alignment

### ISSUE THREAD

**Issue by natxocc** [AT-UI/at-ui#68]
### Steps to reproduce
When you hover on inputNumber component, the buttons appear on right, but if you try to increase o decrease values with these buttons, they appear that invrease button is high than decrease button.
### Which browser?

Chrome 62
### What is expected?
Seems that when hover over these buttons, an effect increase the button selected, but the size on increase and the size on decrease need to be the same.
### What is actually happening?

The size of the buttons on hover are different. Size on increase button are greater that size on decrease button.
### Reproduction Link

Easy to see only in the examples on your page.

**Comment by diego-lipinski-de-castro** [comment]
I though this was on purpose to be honest, lol

**Comment by natxocc** [comment]
Yes, but the sizes are different. When hover on "plus" button the size increase to 50% but when hover on "minus" button increases to 75%. It's only a little error on aspect.
